### ISSUE TITLE
Error on Moon

### ISSUE THREAD

**Issue by Spikebhaal** [sayterdarkwynd/FrackinUniverse#740]
I get an ingame error and i get thrown back to the main menu with my Main Char. I landed on a moon and found a Biome with Scrapblocks and Trees with Eyes after i entered it this error poped up and when i click ok i am in the Main menu. Cant load that char always get the error: "Exception caught in client main-loop (EofException) Failed to read full buffer in readFull, eof reached."

**Comment by brianmose** [comment]
I also encountered this (i think it was on a wasteland planet) 
I fixed it temporarily by deleting the world file of the planet causing the planet to regenerate.
1. Starting from .\Program Files (x86)\Steam\steamapps\common\Starbound\storage\universe
2. Sort the files by date-modified ascending (latest one will be at the top).
3. Delete the latest modified .WORLD file (might be a good idea to make a backup of it, just in case :) )
4. Start playing again!
5. Leave the planet and never return, as the generation is the same as before and could result in another EofException.
